### ISSUE TITLE
JSON output not quite right

### ISSUE THREAD

**Issue by davidmarkclements** [mcollina/pino#25]
```
require('./')().info('%j', {hi: true})
{"pid":14050,"hostname":"MacBook-Pro-3.home","level":30,"msg":"{"hi":true}","time":1459881116335,"v":1}
```
that's illegal JSON (it's actually quick-formats fault), the string will need to be escaped OR we can use pino CLI to sort it out

@mcollina

**Comment by mcollina** [comment]
Yes, the string is wrong, we need to do escaping before putting the message in (in pino, I think).
The problem is here: https://github.com/mcollina/pino/blob/master/pino.js#L115

**Comment by davidmarkclements** [comment]
ah... I just fixed it in quick-format - and published 2.0.1 so any installs of pino should be covered
the fix in quick format doesn't slow down the basic or object cases, it'll possibly hit the interpolation cases by about 20-40ms
